clients/client-transcribe-streaming/package.json

```diff
@@ -49,8 +50,8 @@
     "@aws-sdk/protocol-http": "1.0.0-beta.4",
     "@aws-sdk/region-provider": "1.0.0-beta.4",
     "@aws-sdk/smithy-client": "1.0.0-beta.5",
-    "@aws-sdk/stream-collector-browser": "1.0.0-beta.4",
-    "@aws-sdk/stream-collector-native": "1.0.0-beta.4",
+    "@aws-sdk/stream-collector-browser": "^1.0.0-beta.0",
+    "@aws-sdk/stream-collector-native": "^1.0.0-beta.0",
```

Comment:
Bug?
```suggestion
    "@aws-sdk/stream-collector-browser": "1.0.0-beta.0",
    "@aws-sdk/stream-collector-native": "1.0.0-beta.0",
```

Reply:
Resolved in https://github.com/aws/aws-sdk-js-v3/pull/1082/commits/8e9bbc011160d4bf0fd0e799cf4873a875d95c12